lib/litmus/required.ex

```diff
@@ -3,7 +3,7 @@ defmodule Litmus.Required do
 
   @spec validate(map, String.t(), map) :: {:ok, map} | {:error, String.t()}
   def validate(%{required: true}, field, params) do
-    if Map.has_key?(params, field) do
+    if Map.has_key?(params, field) && params[field] != nil do
```

Comment:
Do you think we should have a separate error for the `nil` case? I think a user might get confused if they're passing in something like `%{ param => nil }`, since *technically* they are passing in `param`, despite it being `nil`.

Reply:
i thought about that but what would the message be? would you use the term `null` instead of `nil` since `nil` is not a JSON thing? I thought i'd avoid the confusion and just say its required. i'm not sure tho

Reply:
Oof, my gut says use `nil` because this is an elixir library, but `null` might be more general.

Reply:
let's leave as-is for now so we can push the decision down the road. if we see a major need for it we can change this later. sgty?